wakuv2/common/filter.go

```diff
@@ -89,28 +98,34 @@ func (fs *Filters) Install(watcher *Filter) (string, error) {
 	}
 
 	watcher.id = id
+
 	fs.watchers[id] = watcher
 	fs.addTopicMatcher(watcher)
+
+	fs.logger.Info("Filters.Install()", zap.String("id", id))
```

Comment:
I've seen a bunch of Info level logs, do you still need them?

Reply:
Nope, thanks for spotting this, fixed.